### PR TITLE
Implement Controller Design

### PR DESCRIPTION
**Pull request by smk4664**
Demo data for Device Controllers in Nautobot. This will be used in demo.nautobot.com to add an example Controller.

### CODE REVIEW

File: jobs/device_controllers/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+"""Initial data required for controllers."""
+from nautobot.apps.jobs import register_jobs
+
+from nautobot_design_builder.design_job import DesignJob
+
+from .context import ControllerDesignContext
+
+
+class ControllerDesign(DesignJob):
+    """Initialize the database with default values needed by the core site designs."""
```
Comment (glennmatthews) [inline jobs/device_controllers/__init__.py:10]: update?

File: jobs/device_controllers/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    has_sensitive_variables = False
+
+    class Meta:
+        """Metadata needed to implement the backbone site design."""
```
Comment (glennmatthews) [inline jobs/device_controllers/__init__.py:14]: update?

**Review by glennmatthews — APPROVED** [review]
LGTM

**Review by abates — APPROVED** [review]
Looks good to me!

**Pull request merged**